cliquet/tests/resource/test_events.py

```diff
@@ -0,0 +1,34 @@
+import webtest
+from pyramid.config import Configurator
+
+from cliquet.events import CollectionChanged
+from cliquet.tests.testapp import main as testapp
+from cliquet.tests.support import unittest, BaseWebTest, get_request_class
+
+
+class EventsTest(BaseWebTest, unittest.TestCase):
+
+    def get_test_app(self, settings=None):
+        settings = self.get_app_settings(settings)
+        self.config = Configurator(settings=settings)
+        app = testapp(config=self.config)
+        app = webtest.TestApp(app)
+        app.RequestClass = get_request_class(self.api_prefix)
+        return app
+
+    def test_events_triggered(self):
+        # let's subcribe to the CollectionChanged event
+        events = []
+
+        def listener(event):
+            events.append(event)
+
+        self.config.add_subscriber(listener, CollectionChanged)
```

Comment:
@leplatrem What is the benefit of using a mock here ? I find the code you're showing is very complicated and convulted compared to the 3 simple lines I have. 

I find mocks very cool when we need to mock an object which methods that might be called by external code, but in the present case it seems overkill unless I am missing something